scroll-animations-1/Overview.bs

```diff
@@ -825,78 +732,66 @@ spec: cssom-view-1; type: dfn;
 			defines an inward offset from the corresponding edge of the scrollport.
 	</dl>
 
-	Note: This property has no effect when combined
-	with a 'view-timeline-attachment' of ''view-timeline-attachment/defer''.
-
-### Attachment of a View Progress Timeline: the 'view-timeline-attachment' property ### {#view-timeline-attachment}
+### View Timeline Shorthand: the 'view-timeline' shorthand ### {#view-timeline-shorthand}
 
-	<pre class='propdef'>
-	Name: view-timeline-attachment
-	Value: [ local | defer | ancestor ]#
-	Initial: local
+	<pre class='propdef shorthand'>
+	Name: view-timeline
+	Value: [ <<'view-timeline-name'>> [ <<'view-timeline-axis'>> ]? ]#
 	Applies to: all elements
-	Inherited: no
-	Computed value: a list of the keywords specified
-	Animation type: not animatable
 	</pre>
 
-	Specifies the [=name attachment=]
-	of each [=view progress timeline=] name
-	specified by 'view-timeline-name'.
+	This property is a [=shorthand=] for setting
+	'view-timeline-name' and 'view-timeline-axis'
+	in a single declaration.
+	It does not set 'view-timeline-inset'.
 
-	Values have the following meanings:
+	ISSUE: Should it reset 'view-timeline-inset' also?
 
-	<dl dfn-type=value dfn-for=view-timeline-attachment>
-		<dt><dfn>local</dfn>
-		<dd>
-			This [=view progress timeline=] name
-			[=attaches=] to the corresponding [=view progress timeline=]
-			defined on this box.
+# Deferred Progress Timelines # {#deferred-timeline}
```

Comment:
Same top-level comment as earlier: I think I'd prefer to cast this as a name-scoping mechanism.

Also, this ultimately needs to go into css-animations-2, not here. Might make sense for us to just draft it there.

Reply:
Yeah, I'm not _actively_ trying to annoy you by going in circles. I was a definitely unsure what to do here, since the issue notes appear to show an unresolved situation where everyone except you resolved on this model https://github.com/w3c/csswg-drafts/issues/7759#issuecomment-1527936739 :upside_down_face:. But after conferring with Tab, they thought writing that model would be fine despite that.

Reply:
I think the version in https://github.com/w3c/csswg-drafts/pull/8906 is functionally saying the same thing, just describing it differently.